docs/topics/mta-rn-upgrade-cli.adoc

```diff
@@ -4,13 +4,13 @@
 
 :_content-type: PROCEDURE
 [id="mta-rn-upgrade-cli_{context}"]
-= Setting the parameter in the {ocp-short} console after upgrading to version {ProductVersion}
+= Setting the parameter in the Tackle Custom Resource (CR) using the CLI after upgrading to version {ProductVersion}
 
-You can set the value of `rwx_supported` in the {ocp-short} console after you upgrade to {ProductShortName} {ProductVersion}.
+You can set the value of `rwx_supported` in the OpenShift CLI after you upgrade to {ProductShortName} {ProductVersion}.
```

Comment:
You can set the value of `rwx_supported` in the Tacle CR using the OpenShift CLI after you upgrade to {ProductShortName} {ProductVersion}.